networking/openshift-sdn/about-openshift-sdn.adoc

```diff
@@ -0,0 +1,27 @@
+[id="about-openshift-sdn"]
+= About OpenShift SDN
+include::modules/common-attributes.adoc[]
+:context: about-openshift-sdn
+
+toc::[]
+
+{product-title} uses a software-defined networking (SDN) approach to provide a
+unified cluster network that enables communication between pods across the
+{product-title} cluster. This pod network is established and maintained by the
+OpenShift SDN, which configures an overlay network using Open vSwitch (OVS).
+
+OpenShift SDN provides three SDN modes for configuring the pod network:
+
+* The _network policy_ mode allows project administrators to configure their own
+isolation policies using `NetworkPolicy` objects. Network policy is the default
+mode in {product-title} {product-version}.
+
+* The _multitenant mode_ provides project-level isolation for pods and services.
```

Comment:
In the other two bullets, you leave "mode" out of the italics - should be consistent.